tests/failpoints/cases/test_region_cache_engine.rs

```diff
@@ -39,10 +50,40 @@ fn must_copr_point_get(cluster: &mut Cluster<ServerCluster>, table: &ProductTabl
         .key_ranges(vec![key_range])
         .build_with(ctx, &[0]);
```

Comment:
```suggestion
        .key_ranges(vec![key_range])
        .start_ts(get_tso(&cluster.pd_client).into())
        .build_with(ctx, &[0]);
```

I forgot to add the latest TSO.